src/array.jl

```diff
@@ -193,18 +193,21 @@ beamform(s, f, sd; fs=framerate(s), method=Bartlett()) = beamform(s, f, 1, sd; f
 
 function beamformer(::Bartlett, R, sv)
   # TODO: add support for shading
-  [abs.(sv[:,j]' * R * sv[:,j]) for j ∈ 1:size(sv,2)]
+  RS = R * sv
+  pseudospectrum = zeros(Float64, size(sv, 2))
+  pseudospectrum .= abs.(dot.(eachcol(sv), eachcol(RS)))
 end
 
 function beamformer(opt::Capon, R, sv)
   opt.γ != 0 && (R .+= opt.γ * I(size(R, 1)))
-  [1 ./ abs.(sv[:,j]' * inv(R) * sv[:,j]) for j ∈ 1:size(sv,2)]
+  pseudospectrum = zeros(Float64, size(sv, 2))
+  RiS = inv(Hermitian(R)) * sv
+  pseudospectrum .= 1 ./ abs.(dot.(eachcol(sv), eachcol(RiS)))
```

Comment:
```suggestion
  1 ./ abs.(dot.(eachcol(sv), eachcol(inv(Hermitian(R)) * sv)))
```

Verification and benchmark:
```julia
julia> 1 ./ abs.(dot.(eachcol(sv), eachcol(inv(Hermitian(R)) * sv))) ≈  [1 ./ abs.(sv[:,j]' * inv(R) * sv[:,j]) for j ∈ 1:size(sv,2)]
true

julia> @btime 1 ./ abs.(dot.(eachcol($sv), eachcol(inv($R) * $sv)));
  59.042 μs (15 allocations: 194.88 KiB)

julia> @btime [1 ./ abs.($sv[:,j]' * inv($R) * $sv[:,j]) for j ∈ 1:size($sv,2)];
  12.500 ms (3843 allocations: 16.61 MiB)
```